src/backend/tests/it/pow.rs

```diff
@@ -200,3 +273,204 @@ fn test_create_pow_challenge_should_fail_if_not_expired() {
         CreateChallengeError::ChallengeInProgress
     );
 }
+
+#[test]
+fn test_allow_signing_should_succeed_with_valid_nonce() {
+    if !helper_is_pow_enabled() {
```

Comment:
It would be nice to make it very visible that this test is being skipped, but I can offer no good way to do this.